<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userInterface</class>
 <widget class="QWidget" name="userInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>381</width>
     <height>550</height>
    </rect>
   </property>
   <property name="title">
    <string>Log</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>300</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>510</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear log screen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>611</width>
     <height>550</height>
    </rect>
   </property>
   <property name="title">
    <string>BlackList</string>
   </property>
   <widget class="QPushButton" name="browseBlacklist">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse blacklist DB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadBlacklist">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>510</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload blacklist DB</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Erase ip from the database</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="deleteIpInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteIpButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete ip from blacklist</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Add ip to the database</string>
    </property>
    <widget class="QPushButton" name="addIpButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add ip to blacklist</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addIpInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDdos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DDOS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DOS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSqlInjection">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sql injection</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableViewBlacklist">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>301</width>
      <height>451</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>741</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>WAF configuration</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dos packets number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dosInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ddosInput">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DDos packets number:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
